src/pages/experiments/[experimentId]/data-exploration/components/heatmap/HeatmapPlot.jsx

```diff
@@ -229,6 +230,14 @@ const HeatmapPlot = (props) => {
     // eslint-disable-next-line no-shadow
     const cartesian = (...a) => a.reduce((a, b) => a.flatMap((d) => b.map((e) => [d, e].flat())));
 
+    // Mapping between expressionValue with key to data
+    const dataDict = {
+      raw: 'expression',
+      zScore: 'zScore',
+    };
+
+    const dataSource = dataDict[expressionValue];
```

Comment:
similarly, I would rename this or just use `dataDict[expressionValue]` directly in line 253